extensions/dub/src/search-links.tsx

```diff
@@ -162,6 +165,12 @@ export function SearchLinks() {
                   onAction={() => deleteLink(id, mutate)}
                 />
                 <ActionPanel.Section>
+                  <Action
+                    title="Show Details"
```

Comment:
```suggestion
                    title="Toggle Details"
```

How about using `Toggle Details` here?

Reply:
Good point! I think I prefer Show / Hide Details though so I'll add that instead...